channels/uk.m3u

```diff
@@ -152,8 +152,10 @@ http://91.218.60.3:8001/play/a00t
 https://liveanevia.mncnow.id/live/eds/BBCWorldNews/sa_dash_vmx/BBCWorldNews.mpd
 #EXTINF:-1 tvg-id="BeanoTV.uk" tvg-name="Beano TV" tvg-country="UK" tvg-language="English" tvg-logo="https://i.imgur.com/XvBi1Ou.png" group-title="",Beano TV (720p)
 https://beanostudios-beanotv-1-gb.samsung.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="BoxHits.uk" tvg-name="Box Hits" tvg-country="UK" tvg-language="English" tvg-logo="" group-title="Music",Box Hits (576p)
-http://csm-e.tm.yospace.com/csm/extlive/boxplus01,boxhits-desktop.m3u8?yo.up=http://boxtv-origin-elb.cds1.yospace.com/uploads/boxhits/
+#EXTINF:-1 tvg-id="BoxHits.uk" tvg-name="Box Hits" tvg-country="UK" tvg-language="English" tvg-logo="https://tvepg.eu/img/united_kingdom/logo/box_hits.png" group-title="Music",Box Hits (576p)
+https://csm-e-boxplus.tls1.yospace.com/csm/extlive/boxplus01,boxhits-alldev.m3u8?yo.up=http://boxtv-origin-elb.cds1.yospace.com/uploads/boxhits/
+#EXTINF:-1 tvg-id="BoxHits.uk" tvg-name="Box Hits" tvg-country="UK" tvg-language="English" tvg-logo="https://tvepg.eu/img/united_kingdom/logo/box_hits.png" group-title="Music",Box Hits (576p) [Geo-blocked]
+https://csm-e-boxplus.tls1.yospace.com/csm/extlive/boxplus01,boxhits-alldev.m3u8?yo.up=https://boxtv.secure.footprint.net/boxhits/
```

Comment:
It's difficult to get the stream from this link.

Reply:
Have you tried a UK VPN? I think MapGuy did

Reply:
On the 3 links above, I didn't need to. It worked directly. I'll try with a UK VPN.

Reply:
Using UK Tor link, it's buffering but doesn't give me anything else.
![image](https://user-images.githubusercontent.com/30985701/124303637-f6deb300-db62-11eb-87a2-54472d45ab20.png)

Reply:
![image](https://user-images.githubusercontent.com/5430116/124303923-b0da1d00-db6b-11eb-8ce7-42eab58f6083.png)

Reply:
Well if that works, that works.